src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,67 @@
-import React from 'react';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import './CurrentUser.scss';
+import '../../styles/general.scss';
+import { getAll } from '../../api';
 
-export const CurrentUser = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+export class CurrentUser extends Component {
+  state = {
+    user: {},
+  }
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+  componentDidMount() {
+    this.loadData();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps !== this.props) {
+      this.loadData();
+    }
+  }
+
+  loadData = async() => {
+    const { userId } = this.props;
+    const prepearedUser = await getAll(`users/${userId}`);
+
+    this.setState({
+      user: prepearedUser.data,
+    });
+  }
+
+  render() {
+    const { user } = this.state;
```

Comment:
Деструктуризации много не бывает) 
```suggestion
    const { id, name, email, phone} = this.state.user ;
```